osu.Game.Rulesets.Catch/Objects/Drawables/DrawableFruit.cs

```diff
@@ -8,8 +8,10 @@
 
 namespace osu.Game.Rulesets.Catch.Objects.Drawables
 {
-    public class DrawableFruit : PalpableDrawableCatchHitObject
+    public class DrawableFruit : DrawablePalpableCatchHitObject
     {
+        public override bool StaysOnPlate => true;
```

Comment:
This override is redundant (restates the `true` from `base`). If we want this to be explicit in each inheritor then the base should be abstract and not virtual.